EventsExpress.Db/Entities/RefreshToken.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace EventsExpress.Db.Entities
+{
+  public  class RefreshToken
+    {
+        public int Id { get; set; }
+        public string Token { get; set; }
+        public DateTime Expires { get; set; }
+        public bool IsExpired => DateTime.UtcNow >= Expires;
```

Comment:
Do we need have any logic in EF entity model? On my mind this should be moved to Core layer DTO model

Reply:
Created RefreshTokenDTO, and deleted method from RefreshTokenEntity

Reply:
created RefreshTokenDTO and moved methods to this model